src/helm/proxy/clients/scale_critique_client.py

```diff
@@ -142,10 +139,20 @@ def _critique_question_to_scale_field(self, question: CritiqueQuestionTemplate,
                 "field_id": question.name,  # This must be unique, so we use the question name
                 "title": question.name,
                 "description": self._interpolate_fields(question.text, fields),
-                "choices": [{"label": option, "value": option} for option in question.options],
+                # Scale's consensus function requires the values to be a number.
+                "choices": [{"label": question.options[i], "value": i} for i in range(len(question.options))],
                 "min_choices": 0 if question.question_type == "checkbox" else 1,
                 "max_choices": len(question.options) if question.question_type == "checkbox" else 1,
             }
+        elif question.question_type == "free_response":
+            return {
+                "type": "text",
+                "field_id": question.name,
+                "title": question.name,
+                "description": question.text,
+                "max_characters": 500,
```

Comment:
500 seems small. Does this affect the size of the text box or something?

Reply:
We do want it to be small, because we just want keyword feedbacks. An example response will be "typos; too long; cuss words"